_studio/shared/umc/io/umc_va/src/umc_va_linux.cpp

```diff
@@ -310,10 +310,10 @@ Status LinuxVideoAccelerator::Init(VideoAcceleratorParams* pInfo)
                                   && ((m_Profile & VA_CODEC) != UMC::VA_H265)
                                   && ((m_Profile & VA_CODEC) != UMC::VA_VP8)
                                   && ((m_Profile & VA_CODEC) != UMC::VA_VP9)
-#ifndef ANDROID
+                                  && ((m_Profile & VA_CODEC) != UMC::VA_VC1)
                                   && ((m_Profile & VA_CODEC) != UMC::VA_MPEG2)
+#ifndef ANDROID
                                   && ((m_Profile & VA_CODEC) != UMC::VA_JPEG)
```

Comment:
What about JPEG? we have adaptive playback support on Android and does not have it on Linux? This looks weird. Can you check whether driver supports this on Linux as well? I would prefer to have no difference between Linux/Android here.

Reply:
needAllocatedSurfaces = true means that Adaptive Playback is unsupported (pre-allocated surfaces are required in this case). So, Adaptive Playback is supported for JPEG on Linux and unsupported on Android. We do not deliver JPEG decoder on Android, so it can be excluded here.